src/contracts/artist_token.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * User: llukac<lukas@lightstreams.io>
+ * Date: 14/10/19 13:40
+ * Copyright 2019 (c) Lightstreams, Granada
+ */
+
+const Web3Wrapper = require('../web3');
+
+const artistTokenSc = require('../../build/contracts/ArtistToken.json');
+const fundingPoolSc = require('../../build/contracts/FundingPoolMock.json');
+
+module.exports.deployFundingPool = async (web3, from) => {
```

Comment:
MINOR: Not consistent with other apis which should be using `{from}` as second argument